wire-ios/Wire-iOS/Sources/UserInterface/Settings/AccentColorPicker.swift

```diff
@@ -38,10 +38,8 @@ struct AccentColorPicker: View {
                 cell(for: color)
                     .listRowBackground(Color(SemanticColors.View.backgroundUserCell))
                     .onTapGesture {
-                        withAnimation {
-                            self.selectedColor = color
-                            onColorSelect?(color)
-                        }
```

Comment:
did you try:
```
withAnimation {
  self.selectedColor = color
}
onColorSelect?(color)
```

Reply:
Yep and it hasn't been successful. Same glitch.

Reply:
ah ok normal I see we change the textStyle so not the same height ...